sympy/functions/special/error_functions.py

```diff
@@ -378,6 +395,14 @@ def _eval_rewrite_as_uppergamma(self, z):
     def _eval_rewrite_as_expint(self, z):
         return S.One - sqrt(z**2)/z + z*expint(S.Half, z**2)/sqrt(S.Pi)
 
+    # TODO: poor style, should inherit/mixin or similar?
+    as_integral = erf.as_integral
+    _eval_rewrite_as_Integral = erf._eval_rewrite_as_Integral
```

Comment:
This isn't very nice.  But IMHO better than copy-paste the same two methods 10 times...  Conrete suggestions welcome or can leave it for another PR?